smtpserver/server.go

```diff
@@ -2418,7 +2418,12 @@ func (c *conn) deliver(ctx context.Context, recvHdrFor func(string) string, msgW
 					m.MessageID = messageid
 					m.MessageHash = messagehash
 					acc.WithWLock(func() {
-						if hasSpace, err := acc.TidyRejectsMailbox(c.log, conf.RejectsMailbox); err != nil {
+						hasSpace := true
+						var err error = nil
```

Comment:
fyi, the `= nil` isn't needed in go. uninitialized values get the zero value. which is nil for the error type (which is an interface).

Reply:
Ah, good point. Done, I think?